src/app/app.component.ts

```diff
@@ -17,23 +20,35 @@ export class WiFiCalling {
 
   pages;
 
-  constructor(platform: Platform, statusBar: StatusBar, splashScreen: SplashScreen, public store: Store<AppState>) {
+  constructor(platform: Platform, translate: TranslateService, statusBar: StatusBar, splashScreen: SplashScreen, public store: Store<AppState>, private globalization: Globalization) {
 
     platform.ready().then(() => {
       statusBar.styleDefault();
       splashScreen.hide();
+
+      translate.setDefaultLang(defaultLang);
+
+      if ((<any>window).cordova) {
```

Comment:
To avoid checking cordova, code can be simplified to
```ts
this.globalization.getPreferredLanguage()
   .then(result => {
        const language = getSuitableLanguage(result.value);
        translate.use(language);
   })
   .catch(err => {
        const browserLanguage = translate.getBrowserLang() || defaultLang;
        const language = getSuitableLanguage(browserLanguage);
        translate.use(language);
    });
}
```